trie/verkle.go

```diff
@@ -248,13 +248,14 @@ func (trie *VerkleTrie) Commit(_ bool) (common.Hash, *NodeSet, error) {
 		return common.Hash{}, nil, fmt.Errorf("serializing tree nodes: %s", err)
 	}
 
+	batch := trie.db.diskdb.NewBatch()
 	for _, node := range nodes {
-		if err := trie.db.diskdb.Put(append([]byte("flat-"), node.Path...), node.SerializedBytes); err != nil {
+		if err := batch.Put(append([]byte("flat-"), node.Path...), node.SerializedBytes); err != nil {
 			return common.Hash{}, nil, fmt.Errorf("put node to disk: %s", err)
 		}
 	}
 
-	return nodes[0].CommitmentBytes, nil, nil
+	return nodes[0].CommitmentBytes, nil, batch.Write()
```

Comment:
Maybe help to apply a similar pattern [as here](https://github.com/gballet/go-ethereum/blob/beverly-hills-head/cmd/utils/cmd.go#L456-L460) (and other places in the codebase) to honor `IdealBatchSize`?

Reply:
makes sense. I will try that and see if the performance improves.